docs/operator-manual/argocd-cm.yaml

```diff
@@ -331,3 +331,43 @@ data:
     - url: https://mycompany.splunk.com?search={{.metadata.namespace}}
       title: Splunk
       if: kind == "Pod" || kind == "Deployment"
+
+  extension.config: |
+    extensions:
+        # Name defines the endpoint that will be used to register
+        # the extension route.
+        # Mandatory field.
+      - name: some-extension
+        backend:
+          # ConnectionTimeout is the maximum amount of time a dial to
+          # the extension server will wait for a connect to complete.
+          # Optional field. Default: 2 seconds
+          connectionTimeout: 2s
+
+          # KeepAlive specifies the interval between keep-alive probes
+          # for an active network connection between the API server and
+          # the extension server.
+          # Optional field. Default: 15 seconds
+          keepAlive: 15s
+
+          # IdleConnectionTimeout is the maximum amount of time an idle
+          # (keep-alive) connection between the API server and the extension
+          # server will remain idle before closing itself.
+          # Optional field. Default: 60 seconds
+          idleConnectionTimeout: 60s
+
+          # MaxIdleConnections controls the maximum number of idle (keep-alive)
+          # connections between the API server and the extension server.
+          # Optional field. Default: 30
+          maxIdleConnections: 30
+
+          services:
+              # URL is the address where the extension backend must be available.
+              # Mandatory field.
+            - url: http://httpbin.org
+
+              # Cluster if provided, will have to match the application
+              # destination name to have requests properly forwarded to this
+              # service URL.
+              # Optional field if only one service is specified. Mandatory if multiple services are specified.
+              cluster: https://some-cluster
```

Comment:
I think this needs to be updated to the new format.

Reply:
fixed